.gitignore

```diff
@@ -2,3 +2,4 @@ _site/
 .sass-cache/
 .jekyll-cache/
 .jekyll-metadata
+_includes/notes_graph.json
```

Comment:
Thanks for adding this entry - it was indeed pretty annoying. 😄

Reply:
Hahaha yep! In general automatically generated files should be ignored, so i'm glad i could make this tiny contribution!